src/patterns/components/inputs/percentage.hbs

```diff
@@ -0,0 +1,34 @@
+---
+name: Percentage Input
+description:
+  This input is used to collect a percent value.
+information:
+  - Spark does not apply any validation or formatting to these fields.
+order: 23
+---
+
+<div class="sprk-b-InputContainer" data-sprk-input="percentage">
+  <div class="sprk-b-TextInputIconContainer sprk-b-TextInputIconContainer--has-text-icon-right">
+    <label for="{{#if idattr}}{{idattr}}{{else}}percentage-normal{{/if}}"
+      class="sprk-b-Label sprk-b-Label--percentage">Interest Rate</label>
```

Comment:
I don't see this class in the Sass anywhere, sprk-b-Label--percentage. What's it applying?